src/specklepy/objects/other.py

```diff
@@ -71,6 +71,19 @@ class DisplayStyle(Base, speckle_type=OTHER + "DisplayStyle"):
     lineweight: float = 0
 
 
+class Text(Base, speckle_type=OTHER + "Text"):
+    """
+    Text object to render it on viewer.
+    """
+    plane: Plane
+    value: str
+    height: float
+    rotation: float
+    displayStyle: DisplayStyle
```

Comment:
Sharp doesn't have this, why is this needed?

Reply:
Because text color retrieves from `DisplayStyle`'s color. Otherwise text cannot be render without `displayValue`. I believe sharp missing this one